src/components/LoanCards/ExpandableLoanCard/expandableLoanCardMixin.js

```diff
@@ -0,0 +1,77 @@
+export default {
```

Comment:
This mixin is placed here since it's just used in `ExpandableLoanCard` and `ExpandableLoanCardCollapsed`